sample_project/Assets/SampleViewer/Samples/Geocoding/Geocoder.cs

```diff
@@ -99,6 +116,22 @@ private void OnLeftClickStart(InputAction.CallbackContext context)
         }
     }
 
+    private void OnTouchInputStarted(InputAction.CallbackContext ctx)
+    {
+        Ray ray = MainCamera.ScreenPointToRay(touchControls.Touch.TouchPosition.ReadValue<Vector2>());
+        if (Physics.Raycast(ray, out RaycastHit hit))
+        {
+            Vector3 direction = (hit.point - MainCamera.transform.position);
+            DistanceFromCamera = Vector3.Distance(MainCamera.transform.position, hit.point);
+            float scale = DistanceFromCamera * LocationMarkerScale / 400000; // Scale the marker based on its distance from camera 
```

Comment:
`400000` should be established as a variable either in this method or in the class. Not good to hide it in code.

Reply:
This doesn't relate to the touch controls added in this PR and can be roped into the General Samples Code Cleanup issue.